styles/global.scss

```diff
@@ -234,7 +234,7 @@ body {
 hr {
   height: 1px;
   @include ktheme() {
-    background: theme('k-grey');
+    background: theme('k-shade');
```

Comment:
won't this break other places?

Reply:
yes I guess, I think it's better to have same colour instead of various colours for dividers. except, we have some variants for dividers. for example `divider.primary`, `divider.accent`

> Dividers
> make them k-shade in light mode, k-hovergrey in dark mode

Reply:
aight